modules/gateways/blockonomics/usdtblockonomics.php

```diff
@@ -0,0 +1,55 @@
+<?php
+require_once __DIR__ . '/../../../init.php';
+require_once __DIR__ . '/../../../includes/gatewayfunctions.php';
+require_once __DIR__ . '/../../../includes/invoicefunctions.php';
+require_once __DIR__ . '/blockonomics.php';
+
+use Blockonomics\Blockonomics;
+
+
+// Init Blockonomics class
+$blockonomics = new Blockonomics();
+$systemUrl = \App::getSystemURL();
+
+function logMessage($message) {
```

Comment:
WHMCS has inbuilt logging feature for modules: [logModuleCall](https://developers.whmcs.com/provisioning-modules/module-logging/)